### ISSUE TITLE
tflite转换 [ DEPTH_TO_SPACE ]算子不支持

### ISSUE THREAD

**Issue by pkjq11** [alibaba/MNN#1452]
# 平台: LINUX

# Github版本: master
# Github Version:

报错情况:
tflite转换时算子不支持，`liteConverter.cpp:239: Check failed: creator ==> NOT_SUPPORTED_OP: [ DEPTH_TO_SPACE ]`

**Comment by pkjq11** [comment]
我仿照SpaceToDepthTflite.cpp在tflite目录下新建了一个DepthToSpaceTflite.cpp
```
#include <stdio.h>
#include "liteOpConverter.hpp"
using namespace tflite;

DECLARE_OP_COVERTER(DepthToSpaceTflite);

MNN::OpType DepthToSpaceTflite::opType(bool quantizedModel) {
    return MNN::OpType_DepthToSpace;
}
MNN::OpParameter DepthToSpaceTflite::type(bool quantizedModel) {
    return MNN::OpParameter_DepthSpaceParam;
}

void DepthToSpaceTflite::run(MNN::OpT* dstOp, const std::unique_ptr<tflite::OperatorT>& tfliteOp,
                        const std::vector<std::unique_ptr<tflite::TensorT>>& tfliteTensors,
                        const std::vector<std::unique_ptr<tflite::BufferT>>& tfliteModelBuffer,
                        const std::vector<std::unique_ptr<tflite::OperatorCodeT>>& tfliteOpSet, bool quantizedModel) {
    auto depthToSpaceParam = new MNN::DepthSpaceParamT;
    auto opt=tfliteOp->builtin_options.AsDepthToSpaceOptions();
    depthToSpaceParam->blockSize = opt->block_size;
    dstOp->main.value = depthToSpaceParam;

}

REGISTER_CONVERTER(DepthToSpaceTflite, BuiltinOperator_DEPTH_TO_SPACE);
```
编译和转换都通过了，我先测试一下模型使用是否正常

**Comment by jxt1234** [comment]
这个是否测试正常？相关 tflite 模型可以发一下?

**Comment by pkjq11** [comment]
> 这个是否测试正常？相关 tflite 模型可以发一下?
这个测试结果是正常的，转换正常，转换后跑出的结果也是正常的，模型我现在不方便导，不过在这个层在tensorflow2的定义是这样的
`up = layers.Lambda(lambda x: tf.nn.depth_to_space(x, 2))(up)`

**Comment by pkjq11** [comment]
> 这个是否测试正常？相关 tflite 模型可以发一下?

这个测试结果是正常的，转换正常，转换后跑出的结果也是正常的，模型我现在不方便导，不过在这个层在tensorflow2的定义是这样的
`up = layers.Lambda(lambda x: tf.nn.depth_to_space(x, 2))(up)`

**Comment by jxt1234** [comment]
好的，我把你的代码加一下

**Comment by jxt1234** [comment]
已经添加

**Comment by CloudGuardian** [comment]
> 时算子不支持，`liteConverter.cpp:239: Check failed: creator ==> NOT_SUPPORTED_OP: [ DEPTH_TO_SPACE ]`

请问我是从pytorch的pixel shuffle和pixel unshuffle转到mnn推理的，转换成功但是精度对不齐，请问pytorch和mnn对这两个算子的实现有差异吗？
